doc/nrf/releases_and_maturity/releases/release-notes-2.5.3.rst

```diff
@@ -0,0 +1,121 @@
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.5.3`_.
+
+IDE and tool support
+********************
+
+`nRF Connect for Visual Studio Code extension <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.5.3.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.5.3_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.5.3_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.5.3`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_253_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+Bluetooth
+---------
+
+Fixed an issue where the Bluetooth host running on the nRF5340 application core might deadlock on disconnection (DRGN-21390).
+
+Wi-Fi
+-----
+
+* Updated WPA supplicant to reserve libc heap memory rather than using leftover RAM.
+  This doesn't affect the overall memory used, but now the RAM footprint as reported by the build will show higher usage.
+
+Matter
+------
+
+* Updated:
+
+  The default Message Reliability Protocol (MRP) retry intervals for Thread devices to two seconds to reduce the number of spurious retransmissions in Thread networks.
+  Increased the number of available packet buffers in the Matter stack to avoid packet allocation issues.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Matter samples
+--------------
+
+* :ref:`matter_lock_sample` sample:
+
+   * Added support for emulation of the nRF7001 Wi-Fi companion IC on the nRF7002 DK.
+
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``a768a05e6205e415564226543cee67559d15b736``.
+
+For a complete list of |NCS| specific commits and cherry-picked commits since v2.5.0, run the following command:
+
```

Comment:
@anangl - Could you please confirm if the zephyr updates are correct?

Reply:
Yes, they are correct. But we should mention somewhere about the QSPI driver fix. I think this could be done here, similarly like the Anomaly 168 workaround was mentioned in 2.5.1, i.e. by adding.: "It also contains some specific additions and commits cherry-picked from the upstream Zephyr repository including the following one:" and then something like:
"- Fixed GPIO configuration that is used in the pinctrl driver for the QSPI IO3 line. The incorrect settings caused that the QSPI NOR flash driver could not initialize successfully for a flash chip configured to work in non-Quad (2 I/O) mode.".

Reply:
Checked. Looks good.